lib/src/inline_parser.dart

```diff
@@ -274,6 +274,118 @@ class AutolinkSyntax extends InlineSyntax {
   }
 }
 
+/// Matches autolinks like `http://foo.com`.
+class AutolinkExtensionSyntax extends InlineSyntax {
+  static const START = r'(?:^|[\s*_~(>])';
```

Comment:
Could you comment on what the static strings are, just as a group.

Reply:
What is the preferred comment style for Dart? I see a lot of `///` elsewhere in the document, but a few `//` as well.

Also, I'll update my other comments to match the preferred style.